src/MediaRental/DatabaseSupport.java

```diff
@@ -465,6 +536,23 @@ public int addCustomer(Customer customer)
                 " VALUES ('" + customer.getName() + "', '" + customer.getAddress() + "');";
         try
         {
+            if(customer.getId()==0)
```

Comment:
Or I could use the large block of code which I already wrote, and works just fine